cmd/wego/version/cmd.go

```diff
@@ -2,42 +2,28 @@ package version
 
 import (
 	"fmt"
-	"os"
 
+	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/weaveworks/go-checkpoint"
 )
 
+// The current wego version
+var Version = "v0.0.0"
```

Comment:
I would rather prefer to have this dynamic. Do can do something like this https://github.com/weaveworks/wksctl/pull/331/files#diff-9099d16d1f5039d08a9c52239830f5355aa0ad958d4f8f18bd5457581acbf546R354

Reply:
@J-Thompson12  I just saw that you had this already. What did we change it ?

Reply:
So we had a conversation at standup yesterday and we need to use checkpoint because it does metrics as well as the version check